src/data/services/LicenseManagerAPIService.js

```diff
@@ -155,6 +155,11 @@ class LicenseManagerApiService {
     return LicenseManagerApiService.apiClient().post(url, options);
   }
 
+  static fetchBulkEnrollmentJob(enterpriseUuid, bulkEnrollmentJobUuid, options) {
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}/bulk-license-enrollment/?enterprise_customer_uuid=${enterpriseUuid}&bulk_enrollment_job_uuid=${bulkEnrollmentJobUuid}`;
```

Comment:
nit: Could use `URLSearchParams` here to construct the query params in the URL instead of doing it manually.

```jsx
const queryParams = new URLSearchParams();
queryParams.append('enterprise_customer_uuid', enterpriseUuid);
queryParams.append('bulk_enrollment_job_uuid', bulkEnrollmentJobUuid);
const url = `${LicenseManagerApiService.licenseManagerBaseUrl}/bulk-license-enrollment/?${queryParams.toString()}`;
```